src/classes/VOL_CTRL_PersonalSiteTemplate_TEST.cls

```diff
@@ -33,7 +33,7 @@ public with sharing class VOL_CTRL_PersonalSiteTemplate_TEST {
 
     //==================== TEST METHOD(s) ======================================
     
-    @isTest(SeeAllData=true)
+    @isTest(SeeAllData=true) // SeeAllData so the CSS file is viewable
     public static void TestPersonalSiteTemplate() {
```

Comment:
Can this test method be private instead of public? 

This might be applicable for all test methods introduced in this PR.